10-frontdoor.md

```diff
@@ -12,22 +12,21 @@ We will reach two goals:
 1. Read the FQDN values from each Azure Application Gateway
 
 ```bash
-APPGW_FQDN_BU0001A0042_03=$(az deployment group show -g rg-bu0001a0042-03 -n  spoke-BU0001A0042-04 --query properties.outputs.appGwFqdn.value -o tsv)
-APPGW_FQDN_BU0001A0042_04=$(az deployment group show -g rg-bu0001a0042-04 -n spoke-BU0001A0042-04 --query properties.outputs.appGwFqdn.value -o tsv)
-
+export APPGW_FQDN_BU0001A0042_03=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0042-03 --query properties.outputs.appGwFqdn.value -o tsv)
+export APPGW_FQDN_BU0001A0042_04=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0042-04 --query properties.outputs.appGwFqdn.value -o tsv)
 ```
 
 1. Create resource group in order to deploy Azure Front Door
 
 ```bash
-az group create --name rg-global-front-door --location eastus2
+az group create --name rg-bu0001a0042-global --location centralus
```

Comment:
Please, change the rg name in the cleanup file too.

Reply:
done